src/main/java/sirius/db/mixing/types/MultiLanguageString.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.db.mixing.types;
+
+import sirius.kernel.Sirius;
+import sirius.kernel.health.Exceptions;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Provides a language-text map as property value.
+ * <p>
+ * These are stored in MongoDB as an array containing sub-documents
+ * containing a <tt>lang</tt> and <tt>text</tt> properties
+ *
+ * @see sirius.db.mixing.properties.MultiLanguageStringProperty
+ */
+public class MultiLanguageString extends SafeMap<String, String> {
```

Comment:
I'm kinda missing a 'getText' method without parameters that returns NLS.getCurrentLanguage and falls back to the default language

Reply:
i'd also like this in favor of defaultLanguage